systems-manager/documentation/AWS-Systems-Manager-Intro.md

```diff
@@ -246,6 +246,30 @@ The example contains the directories CS_AMAZON and CS_WINDOWS. Each directory sh
 * Uninstall script
 * Sensor Binary for the particular OS
 
+The Install Script At the top of the install script enter values for the the following
+filename=<<filename of the rpm package>>
+TARGET_VER=<<minimum version>> example format =5.38.10404
```

Comment:
```suggestion
Update the following variables at the top of the installation script:

```
filename=<<filename of the rpm package>>
TARGET_VER=<<minimum version>> example format =5.38.10404
```
```